signing/secp256k1/impl/src/main/java/tech/pegasys/signers/secp256k1/multikey/SigningMetadataTomlConfigLoader.java

```diff
@@ -45,20 +46,30 @@
   private static final String GLOB_CONFIG_MATCHER = "**" + CONFIG_FILE_EXTENSION;
 
   private final Path tomlConfigsDirectory;
+  private final Function<String, String> identityFilenameTranslator;
 
-  public SigningMetadataTomlConfigLoader(final Path rootDirectory) {
+  public SigningMetadataTomlConfigLoader(
+      final Path rootDirectory, final Function<String, String> identityFilenameTranslator) {
```

Comment:
nit: Perhaps an interface so would be nicer than Function<String, String> so we can give this type a name